src/systems/joint_position_controller/JointPositionController.cc

```diff
@@ -190,9 +202,26 @@ void JointPositionController::Configure(const Entity &_entity,
     this->dataPtr->jointPosCmd = _sdf->Get<double>("initial_position");
   }
 
+  if (_sdf->HasElement("use_actuator_msg") &&
+    _sdf->Get<bool>("use_actuator_msg"))
+  {
+    if (_sdf->HasElement("actuatorNumber"))
+    {
+      this->dataPtr->actuatorNumber =
+        _sdf->Get<int>("actuatorNumber");
+      this->dataPtr->useActuatorMsg = true;
+    }
+    else
+    {
+      gzerr << "Please specify an actuatorNumber" <<
+        "to use Actuator position message control." << std::endl;
+    }
+  }
+
```

Comment:
I don't know if I caught this in the other PR, but we do underscores in sdf tags:

```suggestion
  if (_sdf->HasElement("use_actuator_msg") &&
    _sdf->Get<bool>("use_actuator_msg"))
  {
    if (_sdf->HasElement("actuator_number"))
    {
      this->dataPtr->actuatorNumber =
        _sdf->Get<int>("actuator_number");
      this->dataPtr->useActuatorMsg = true;
    }
    else
    {
      gzerr << "Please specify an actuator_number" <<
        "to use Actuator position message control." << std::endl;
    }
  }

```

Reply:
@mjcarroll Ok, I can update all of them easily, I was following the style that I had seen from:

https://github.com/gazebosim/gz-sim/blob/14d583283a860efa71d80d9d6d620ba0b31afa97/src/systems/multicopter_motor_model/MulticopterMotorModel.cc#L286-L290

Should I change it in multicopter_motor_model to match as well?

Reply:
Here is my fix for the previous ones: https://github.com/gazebosim/gz-sim/pull/1976